packages/cli/lib/projects.js

```diff
@@ -238,6 +238,7 @@ const ensureProjectExists = async (
     allowCreate = true,
     noLogs = false,
     withPolling = false,
+    uploadCommand = false,
```

Comment:
hmmm the arguments here are getting a little out of hand, but I see why you added this. Seems fine to me for now 👍